### ISSUE TITLE
Ubuntu Server Ubuntu 14.04.2 LTS upgrade to 2.0 fails

### ISSUE THREAD

**Issue by Nogbit** [rethinkdb/rethinkdb#4086]
Doing ``sudo apt-get upgrade rethinkdb`` I get the following 404 from download.rethinkdb.com

```
Do you want to continue? [Y/n] y
Err http://download.rethinkdb.com/apt/ trusty/main rethinkdb amd64 2.0.1~0trusty
  404  Not Found
E: Failed to fetch http://download.rethinkdb.com/apt/pool/trusty/main/r/rethinkdb/rethinkdb_2.0.1~0trusty_amd64.deb  404  Not Found

E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```

**Comment by AtnNn** [comment]
Please try again in a few minutes. I am currently uploading the packages for the new 2.0.1 release and our upload process is not atomic. Sorry for the inconvenience.

**Comment by danielmewes** [comment]
@Nogbit the package (RethinkDB 2.0.1, a bug fix release over 2.0.0) should be up now. Can you retry the upgrade?

**Comment by Nogbit** [comment]
That worked, thanks!
